composer.json

```diff
@@ -17,16 +17,16 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.2",
-        "symfony/framework-bundle": ">=2.3.0",
-        "symfony/yaml": "~2.2",
-        "symfony/locale": "~2.2",
-        "symfony/validator": "~2.2",
+        "symfony/framework-bundle": "~3.0",
```

Comment:
this will make the bundle work with symfony 3 only, and no longer with any 2.x version. i think #179 should be preferred.